08-collab.md

```diff
@@ -97,7 +97,28 @@ Fast-forward
  1 file changed, 1 insertion(+)
  create mode 100644 pluto.txt
 ~~~
-
+>
+> ## Review of the Basic Collaborative Workflow using Git {.callout}
+>
+> A quick review of the basic collaborative workflow using is:
+> * track a change to the repository with `git add`
+> * stage the change using `git commit -m`
+> * upload the changes to GitHub with `git push origin master`
+> * downloand changes to our local copy of the repository using the `git push origin master`
```

Comment:
There is a typo here `git push origin master` should be `git pull origin master`. 

Also, do you think it would be a bit clear if you split the first three steps under an item, such as "Adding your changes to the common repo" and the fourth under another item, such as "Getting other people's changes from the common repo"? With your current wording it may seem that you should pull after pushing.